static/docs/get-started/initialize.md

```diff
@@ -1,12 +1,15 @@
 # Initialize
 
-In order to start using DVC, you need first to initialize it in your
-<abbr>workspace</abbr>. DVC doesn't require Git and can work without any source
-control management system, but for the best experience we recommend using DVC on
-top of Git repositories.
+First of all install DVC from `pip` (on the [installation](/doc/install) page
+there are other options and details):
 
-If you don't have a directory for this <abbr>project</abbr> already, create it
-now with these commands:
+```dvc
+$ pip install dvc
+```
+
+Then create a <abbr>workspace</abbr>. DVC doesn't require Git and can work
```

Comment:
I think we can omit the remark about SCM systems to be honest. Or simplify it to "For the best experience we
recommend using DVC on top of Git repositories."